pyocd/tools/lists.py

```diff
@@ -107,15 +106,8 @@ def list_targets(pack):
                 'part_families' : t.part_families,
                 'part_number' : t.part_number,
                 }
-            if t._svd_location is not None and IS_CMSIS_SVD_AVAILABLE:
-                if t._svd_location.is_local:
-                    svdPath = t._svd_location.filename
-                else:
-                    resource = "data/{vendor}/{filename}".format(
-                        vendor=t._svd_location.vendor,
-                        filename=t._svd_location.filename
-                    )
-                    svdPath = pkg_resources.resource_filename("cmsis_svd", resource)
+            if t._svd_location is not None:
+                svdPath = t._svd_location.filename
                 if isinstance(svdPath, six.string_types) and os.path.exists(svdPath):
                     d['svd_path'] = svdPath
```

Comment:
With SVDs only being in Packs now, we can probably just remove the `svd_path` key since there won't be a simple filename. Alternatively, we could use a format like "<absolute-pack-path>;<svd-path-within-pack>".